rotkehlchen/chain/ethereum/decoding/pickle/constants.py

```diff
@@ -0,0 +1,101 @@
+PICKLE_WBTC_ETH_SUSHI = string_to_ethereum_address('0xde74b6c547bd574c3527316a2eE30cd8F6041525')
+PICKLE_YFI_ETH = string_to_ethereum_address('0x3261D9408604CC8607b687980D40135aFA26FfED')
+PICKLE_BAC_DAI = string_to_ethereum_address('0x4Cac56929B98d4C52dDfDF998329622013Fed2a5')
+PICKLE_MIC_USDT = string_to_ethereum_address('0xC66583Dd4E25b3cfc8D881F6DbaD8288C7f5Fd30')
+PICKLE_MIS_USDT = string_to_ethereum_address('0x0FAA189afE8aE97dE1d2F01E471297678842146d')
+PICKLE_YVECRV_ETH = string_to_ethereum_address('0x5Eff6d166D66BacBC1BF52E2C54dD391AE6b1f48')
+PICKLE_BAS_DAI = string_to_ethereum_address('0xcF45563514a24b10563aC0c9fECCd3476b00DF45')
+PICKLE_MIR_UST = string_to_ethereum_address('0x3Bcd97dCA7b1CED292687c97702725F37af01CaC')
+PICKLE_MTSLA = string_to_ethereum_address('0xaFB2FE266c215B5aAe9c4a9DaDC325cC7a497230')
+PICKLE_MAAPL = string_to_ethereum_address('0xF303B35D5bCb4d9ED20fB122F5E268211dEc0EBd')
+PICKLE_MQQQ = string_to_ethereum_address('0x7C8de3eE2244207A54b57f45286c9eE1465fee9f')
+PICKLE_MSLV = string_to_ethereum_address('0x1ed1fD33b62bEa268e527A622108fe0eE0104C07')
+PICKLE_MBABA = string_to_ethereum_address('0x1CF137F651D8f0A4009deD168B442ea2E870323A')
+PICKLE_ETH_SUSHI = string_to_ethereum_address('0xECb520217DccC712448338B0BB9b08Ce75AD61AE')
+PICKLE_FEI_TRIBE = string_to_ethereum_address('0xC1513C1b0B359Bc5aCF7b772100061217838768B')
+PICKLE_SADDLE_D4 = string_to_ethereum_address('0xe6487033F5C8e2b4726AF54CA1449FEC18Bd1484')
+PICKLE_LUSD_ETH = string_to_ethereum_address('0x927e3bCBD329e89A8765B52950861482f0B227c4')
+PICKLE_ALCX_ETH = string_to_ethereum_address('0x9eb0aAd5Bb943D3b2F7603Deb772faa35f60aDF9')
+PICKLE_YVBOOST_ETH = string_to_ethereum_address('0xCeD67a187b923F0E5ebcc77C7f2F7da20099e378')
+PICKLE_CVX_ETH = string_to_ethereum_address('0xDCfAE44244B3fABb5b351b01Dc9f050E589cF24F')
+PICKLE_RLY_ETH = string_to_ethereum_address('0x0989a227E7c50311f7De61e5e61F7c28Df8936f0')
+PICKLE_YEARN_USDC = string_to_ethereum_address('0xEB801AB73E9A2A482aA48CaCA13B1954028F4c94')
+PICKLE_YEARN_LUSD_3CRV = string_to_ethereum_address('0x4fFe73Cf2EEf5E8C8E0E10160bCe440a029166D2')
+PICKLE_FRAX_3CRV = string_to_ethereum_address('0xd632f22692FaC7611d2AA1C0D552930D43CAEd3B')
+PICKLE_IRONBANK = string_to_ethereum_address('0x4E9806345fb39FFebd70A01f177A675805019ba8')
+PICKLE_MIM_3CRV = string_to_ethereum_address('0x1Bf62aCb8603Ef7F3A0DFAF79b25202fe1FAEE06')
+PICKLE_FOX_ETH = string_to_ethereum_address('0xeb8174F94FDAcCB099422d9A816B8E17d5e393E3')
+PICKLE_CVXCRV_CRV = string_to_ethereum_address('0xF1478A8387C449c55708a3ec11c143c35daf5E74')
+PICKLE_CVXCRV = string_to_ethereum_address('0xB245280Fd1795f5068DEf8E8f32DB7846b030b2B')
+PICKLE_TRU_ETH = string_to_ethereum_address('0x1d92e1702D7054f74eAC3a9569AeB87FC93e101D')
+PICKLE_NEWO_USDC = string_to_ethereum_address('0xBc57294Fc20bD23983dB598fa6B3f306aA1a414f')
+PICKLE_LOOKS_ETH = string_to_ethereum_address('0x69CC22B240bdcDf4A33c7B3D04a660D4cF714370')
```

Comment:
that's a lot of constants! Isn't there a registry or something?

Reply:
No, there is no registry. They were taken from https://github.com/pickle-finance/contracts . Next step will be to add them as assets with protocol and refactor this to get them from the globaldb